### ISSUE TITLE
Clash between signal.clean() and DVARS calculation

### ISSUE THREAD

**Issue by mibur1** [nilearn/nilearn#2648]
Hi,
When signal.clean() is used for confound regression and bandpass filtering, the results are automatically z-standardized. I then don't really see how to calculate DVARS from this. Could it be a possibility with the current calculations to allow a standardization=False for this?

Corresponding neurostars post: https://neurostars.org/t/calculating-dvars-after-confound-regression-with-nileran/17843

Thanks!

**Comment by NicolasGensollen** [comment]
Hi @mibur1 

Do you have an example we can reproduce where you observe this behaviour? Looking at the code I don't think the signal gets z-standardized if you set `standardize` to False. 
The only thing I can think of is that the function `signal.clean()` was modified a few weeks ago to support a `standardize_confounds` argument (see #2609) such that you can standardize signals and confounds separately instead of relying on the same argument for both (which was indeed causing problems...). 
The fix is not included in Nilearn `0.7.0`, so you'd have to install nilearn from source to use this option.

So if you run:

```python
nilearn.signal.clean(ts, detrend=False, 
                         standardize=False, 
                         standardize_confounds=True,
                         confounds=confounds)
```

the signal shouldn't be z-standardized. 

Here is an example:

```python
import numpy as np
from nilearn.input_data import NiftiMasker
from nibabel import Nifti1Image
np.random.seed(42)

def _simu_img():
    # Random confounds
    conf = 2 + np.random.randn(100, 6)
    # Random 4D volume
    vol = 100 + 10 * np.random.randn(5, 5, 2, 100)
    img = Nifti1Image(vol, np.eye(4))
    # Create an nifti image with the data, and corresponding mask
    mask = Nifti1Image(np.ones([5, 5, 2]), np.eye(4))
    return img, mask, conf

img, mask, conf = _simu_img()
standardize_signal = False
standardize_confounds = True
masker = NiftiMasker(standardize=standardize_signal,
                         standardize_confounds=standardize_confounds,
                         detrend=False,
                         mask_img=mask).fit()
tseries = masker.transform(img, confounds=conf)
print(np.mean(tseries[:,0]))
print(np.std(tseries[:,0]))
```

Output:

```
100.24093863364534
10.343520376862132
```

**Comment by mibur1** [comment]
Hi, thanks for the swift answer! I was indeed using Nilearn 0.7.0, which seems to be the issue. 
I'm not able to check my data at the moment, but using your example code with the latest version from github provides the expected, unstandardized timeseries.

Thanks!
